cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
If this is a FF bug (perhaps it is), why are we only now seeing it after the jQuery upgrade?

I made a JSFiddle using vanilla JS/standard Web APIs which properly detects focus, including in FF: https://jsfiddle.net/neoru0tt/26/ If all it takes to keep the test alive is to swap out jQuery for plain JS, then we should just do that.

I'd really regret to see us remove or disable tests for learner value, especially when that value is an essential accessibility behavior like focus management which is rarely tested.